src/main/java/io/github/jhipster/online/util/SanitizeInputs.java

```diff
@@ -20,13 +20,29 @@
 package io.github.jhipster.online.util;
 
 public class SanitizeInputs {
+    private static final String SANITIZE_REGEX = "[\n\r\t]";
+    private static final String ALPHANUMERIC_REGEX = "[a-zA-Z0-9]*";
```

Comment:
The problem is still the same: the pattern is computed within String.matches(), the type of the constant should be Pattern not String.

https://www.baeldung.com/java-regex-pre-compile

Reply:
Ops you are right. I think I misread your second comment somehow. Thanks for pointing it out. 😄 

I've corrected this at https://github.com/jhipster/jhipster-online/pull/216